tasks/noci_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+EXIT_STATE=0
+
+# tests that aren't run on CI
+
+# jasmine specs with @noCI tag
+npm run citest-jasmine --  tests/*_test.js --tags noCI || EXIT_STATE=$?
```

Comment:
I'm very pleased with [`karma-jasmine-spec-tags`](https://github.com/mnasyrov/karma-jasmine-spec-tags). This :arrow_double_up: allow us to run allow the specs with the `@noCI` tag. This will make the pre-release process a lot less painful.